openshift-install-powervs

```diff
@@ -1080,6 +1080,39 @@ function variables {
     fi
   fi
 
+  question "Do you want to use IBM Cloud Classic DNS and VPC Load Balancer services?" "yes no"
+  if [ "${value}" == "yes" ]; then
+	question "Enter IBM Cloud VPC name"
+	if [[ "${value}" != "" ]]; then
+	  echo "ibm_cloud_vpc_name = \"${value}\"" >> $VAR_TEMPLATE
+	else
+	  error "IBM Cloud VPC name must be non-empty for using IBM Cloud Services"
+	fi
+
+	question "Enter IBM Cloud VPC subnet name"
+	if [[ "${value}" != "" ]]; then
+	  echo "ibm_cloud_vpc_subnet_name = \"${value}\"" >> $VAR_TEMPLATE
+	else
+	  error "IBM Cloud VPC subnet name must be non-empty for using IBM Cloud Services"
+	fi
+
+	debug_switch
+	question "Enter IBM Cloud Classic username"
+	if [[ "${value}" != "" ]]; then
+	  echo "iaas_classic_username = \"${value}\"" >> $VAR_TEMPLATE
+	  question "Enter IBM Cloud Classic API Key for above username" "-sensitive"
+	  if [[ "${value}" != "" ]]; then
+		echo "iaas_classic_api_key = \"${value}\"" >> $VAR_TEMPLATE
```

Comment:
Can we also take the Classic API Key from the environment variable like the existing approach ?
Also I was wondering do we really need the classic username and the key both ?

cc @yussufsh

Reply:
Yes @bpradipt will update for environment variable I/p (possibly will need some additional checks)